ng/src/web/components/table/stripedtable.js

```diff
@@ -43,7 +44,42 @@ const StripedTable = glamorous(Table)({
       background: '#DDDDDD',
     },
   },
-});
+},
+  ({doubleRow}) => {
+    if (doubleRow) {
+      return {
+        // FIXME when details are toggled, coloring is scrambled up, due to
+        // adding just one more tr and messing with the groups of two
+        '@media screen': {
```

Comment:
Despite the issue described in the comment for line 71:
The coloring of groups of rows breaks, once the details for an entity are toggled. As the details are effectively added as a single tr, grouping with even/odd doesn't work anymore. A simple, yet "ugly" solution would be to add an empty table row with each toggled details row.
So we would need a reset of the row counter right after toggled details or another way of selecting the correct rows. Right now, I don't have a solution for this, but I wanted to show the changes already and maybe get some feedback on how to fix this.

Reply:
Or we go back to the rows each wrapped in a tbody. In that case the details row should also get a tbody around it, which should solve all problems.